style.css

```diff
@@ -45,27 +46,67 @@ html,
 }
 
 .button-list {
-  display: flex;
-  align-items: center;
-  width: 58%;
-  justify-content: flex-end;
+  position: fixed;
+  z-index: 100;
+  flex-direction: column;
+  top: 0;
+  gap: 0;
+  align-items: baseline;
+  justify-content: flex-start;
   margin-bottom: 0;
   background-color: #3c3a39;
+  color: #fff5e1;
   font-family: 'Inter', sans-serif;
+  line-height: 44px;
+  height: 100%;
+  width: 100vw;
+  display: none;
+  transition: 0.3s;
+}
+
+.button-list-bottom {
+  position: absolute;
+  top: 90vh;
+  background-color: #fff;
+  width: 100vw;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 10vh;
+}
```

Comment:
- [ ] One of the requirements is to make your design look very identical to that of the Figma template that you chose. For the mobile menu design, the bottom part is not identical to that of the mobile menu of the Figma template that you chose. Kindly make the design identical all through. 👍 💻 

### Your Design

![image](https://github.com/eamador1/portfolio-project/assets/77106962/e763924c-e3d9-40fc-9640-af16cc0d4969)

### Figma Template

![image](https://github.com/eamador1/portfolio-project/assets/77106962/471d2dfc-01e4-43e4-a746-33d8bdbf3b9d)

Reply:
@Gambit142 if you see the figma design, it looks like the bottom that I included is part of the figma design. You can also argue that it is the bottom of a phone which does not make much sense to me., but it is ok👍👍